ios/emerald_mobile_walletTests/emerald_mobile_walletTests.m

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
```

Comment:
maybe remove these tests. i dont think we will need them

Reply:
I can remove the ios tests... you're right - we'd probably use jasmine or jest